packages/validator/src/lib/psi/PsiResult.ts

```diff
@@ -30,6 +30,20 @@ export type LighthouseCategories = {
   [key in LighthouseCategoryName]: LighthouseCategory;
 };
 
+export type LighthouseAuditDetailsItem = {
```

Comment:
I'm wondering if there's a better name for this object - something like `AuditResults` maybe? "Item" implies that there's only one thing so it's a bit odd to have a map which is called XxxItem.

Reply:
changed.